build.gradle

```diff
@@ -54,7 +54,7 @@ ext.deps = [
     javapoet: 'com.squareup:javapoet:1.6.1',
 
     // Other
-    rxjava: 'io.reactivex:rxjava:1.1.2',
+    rxjava: 'io.reactivex:rxjava:1.1.4',
```

Comment:
Hello I am from the future!

On Sat, Apr 23, 2016 at 3:04 AM Zac Sweers <notifications@github.com> wrote:

> In build.gradle
> <https://github.com/JakeWharton/butterknife/pull/487#discussion_r60824433>
> :
>
> > @@ -54,7 +54,7 @@ ext.deps = [
> >      javapoet: 'com.squareup:javapoet:1.6.1',
> >
> >      // Other
> > -    rxjava: 'io.reactivex:rxjava:1.1.2',
> > +    rxjava: 'io.reactivex:rxjava:1.1.4',
>
> Am I missing something or shouldn't this be 1.1.3?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JakeWharton/butterknife/pull/487/files/fc6aa6a29e5bfb9b2d97bc21b153c5d04c07720c#r60824433>
>